examples/cube/src/main.rs

```diff
@@ -1,7 +1,7 @@
-use glam::UVec2;
+use glam::{Mat3A, Mat4, UVec2, Vec3, Vec3A};
 
-fn vertex(pos: [f32; 3]) -> glam::Vec3 {
-    glam::Vec3::from(pos)
+fn vertex(pos: [f32; 3]) -> Vec3 {
+    Vec3::from(pos)
```

Comment:
Uhh, I think these were vscode being helpful with the imports

Reply:
We should always fully import all types in the examples, so it's clear where things are coming from. The Use that was there before was accidental.

Reply:
alright, makes sense

Reply:
Uhh, even for `scene-viewer`? It has loads of use imports.